src/main/java/uk/gov/hmcts/reform/sscs/transformers/SscsCaseTransformer.java

```diff
@@ -365,20 +366,24 @@ private String transformTitle(String title) {
     }
 
     private Address buildPersonAddress(Map<String, Object> pairs, String personType) {
-        if (findBooleanExists(getField(pairs, personType + "_address_line4"))) {
+        if (findBooleanExists(getField(pairs, personType + ADDRESS_LINE4))) {
             return Address.builder()
-                .line1(getField(pairs, personType + "_address_line1"))
-                .line2(getField(pairs, personType + "_address_line2"))
-                .town(getField(pairs, personType + "_address_line3"))
-                .county(getField(pairs, personType + "_address_line4"))
-                .postcode(getField(pairs, personType + "_postcode"))
+                .line1(getField(pairs, personType + ADDRESS_LINE1))
+                .line2(getField(pairs, personType + ADDRESS_LINE2))
+                .town(getField(pairs, personType + ADDRESS_LINE3))
+                .county(getField(pairs, personType + ADDRESS_LINE4))
+                .postcode(getField(pairs, personType + ADDRESS_POSTCODE))
                 .build();
         }
+        boolean line3IsBlank = false;
+        if (!ObjectUtils.isEmpty(getField(pairs, personType + ADDRESS_LINE2)) && ObjectUtils.isEmpty(getField(pairs, personType + ADDRESS_LINE3))) {
```

Comment:
Would using the `findBooleanExists` method instead of isEmpty be better to be consistent with the block just above this change (line 369?). Up to you...

Reply:
Ok thanks will do